src/Symfony/Component/Routing/Loader/AnnotationFileLoader.php

```diff
@@ -92,6 +92,11 @@ protected function findClass($file)
         $class = false;
         $namespace = false;
         $tokens = token_get_all(file_get_contents($file));
+
+        if (1 === count($tokens) && T_INLINE_HTML === $tokens[0][0]) {
```

Comment:
this is because the PHP start tag is either ``<?php `` or ``<?phpEOL`` (with whatever supported EOL char)